### PR TITLE
new icon: trpc (original, original-wordmark, plain, plain-wordmark)

### LINKED ISSUE #1732: [ICON REQUEST]: tRPC
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

tRPC stands for TypeScript Remote Procedure Call, and is the most simple and lightweight library for remotely calling backend functions on the client side. 

### Links and sources

1. [official website](https://trpc.io/)
2. [logo](https://trpc.io/img/logo.svg)

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by sinuheshinbr**
## Double check these details before you open a PR

- [X] PR does not match another non-stale PR currently opened
- [X] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [X] PR's base is the `develop` branch.
- [X] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [X] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [X] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1732**

## Link to prove your SVG is correct and up-to-date.
https://trpc.io/

### CODE REVIEW

**Review by lunatic-fox — CHANGES_REQUESTED** [review]
Hello there! 👋🏼
The icons are pretty good, but they are different from the official logos, please consider to use the listed below to base the new icons.

`original`
<table align="center">
  <tr>
    <td>
      
    </td>
  </tr>
</table>



`original-wordmark`
<table align="center">
  <tr>
    <td>
      
    </td>
    <td>
       
    </td>
  </tr>
</table>

> *Remember to choose only one `original-wordmark` version.*

With this change `plain` and `plain-wordmark` will need to be new monocolor icons.

In `devicon.json` the `versions` property should be like:
```json
        "versions": {
            "svg": [
                "original",
                "original-wordmark",
                "plain",
                "plain-wordmark"
            ],
            "font": [
                "plain",
                "plain-wordmark"
            ]
        },
```

**Comment by lunatic-fox** [comment]
Thank you for the fast response! 🚀

File: devicon.json [hunk 1]
```diff
@@ -9247,6 +9247,29 @@
         "color": "#23719f",
         "aliases": []
     },
+    {
+        "name": "trpc",
+        "altnames": [],
```
Comment (Snailedlt) [inline devicon.json:9252]:
```suggestion
        "altnames": [
            "Typescript Remote Procedure Call"
        ],
```

File: devicon.json [hunk 2]
```diff
@@ -9247,6 +9247,29 @@
...
+        "tags": [
+            "typescript",
```
Comment (Snailedlt) [inline devicon.json:9254]:
```suggestion
            "typescript",
            "javascript",
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Almost there. Just a few suggestions :)

**Comment by lunatic-fox** [comment]
Sorry, I made a little mistake on icon name. Fixed!

**Comment by Panquesito7** [comment]
It appears the screenshot won't be sent due that the workflow is being taken from the `master` branch, and the workflows are still using the Ubuntu 18.04 image, which is no longer supported in GitHub Actions.
We can manually upload these icons to Icomoon and see the results, though.

https://github.com/devicons/devicon/actions/runs/4832860711

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🚀

image
image

P.S.: we will need to make a release ASAP.

**Comment by lunatic-fox** [comment]
> It appears the screenshot won't be sent due that the workflow is being taken from the `master` branch, and the workflows are still using the Ubuntu 18.04 image, which is no longer supported in GitHub Actions. We can manually upload these icons to Icomoon and see the results, though.
> 
> https://github.com/devicons/devicon/actions/runs/4832860711

Yes, I've make a note on #1731.

**Review by lunatic-fox — APPROVED** [review]
Great work! Everything looks fine! ✔
Thank you for this contribution! 🚀

**Pull request merged**